components/bl-transitions-component/src/collapse-left.js

```diff
@@ -53,7 +56,7 @@ export function CollapseLeft(props) {
 
   useEffect(() => {
     if (element.current) {
-      element.current.classList.add('transition', variants);
+      element.current.classList.add('transition', variant);
```

Comment:
What does `element`? 'element` is pretty much the same as `data`, it could be completely everything and doesn't describe anything at the same time. Please, choose a meaningful name or explain why we choose `element` name.